### PR TITLE
Fixes HTTP verbs for workbook range functions

### PR DESCRIPTION
**Pull request by zengin**
- Fixed workbook range functions to use GET to get ranges.
- Regenerated language snippets

- This fixes 4 Beta and 4 V1 C# tests for Raptor.

### CODE REVIEW

File: api-reference/beta/api/workbookrange-rowsabove.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ One of the following permissions is required to call this API. To learn more, in
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
```
Comment (baywet) [inline api-reference/beta/api/workbookrange-rowsabove.md:29]:
```suggestion
```msgraph-interactive
```

File: api-reference/beta/api/workbookrange-columnsafter.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ One of the following permissions is required to call this API. To learn more, in
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
```
Comment (baywet) [inline api-reference/beta/api/workbookrange-columnsafter.md:29]:
```suggestion
```msgraph-interactive
```

File: api-reference/beta/api/workbookrange-columnsbefore.md [hunk 3]
```diff
@@ -27,7 +27,7 @@ One of the following permissions is required to call this API. To learn more, in
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
```
Comment (baywet) [inline api-reference/beta/api/workbookrange-columnsbefore.md:29]:
```suggestion
```msgraph-interactive
```

File: api-reference/beta/api/workbookrange-rowsbelow.md [hunk 4]
```diff
@@ -27,7 +27,7 @@ One of the following permissions is required to call this API. To learn more, in
 ## HTTP request
 <!-- { "blockType": "ignored" } -->
 ```http
```
Comment (baywet) [inline api-reference/beta/api/workbookrange-rowsbelow.md:29]:
```suggestion
```msgraph-interactive
```

**Review by baywet — COMMENTED** [review]
Isn't it missing msgraph-interactive instead of http at a couple of places?

**Comment by zengin** [comment]
@baywet, those are the definitions of requests at the top of documentation, not part of the examples. I think we shouldn't mark them interactive.

**Comment by baywet** [comment]
@zengin ah ok thanks for the distinction! I was never clear on the different block types in the documentation

**Review by baywet — APPROVED** [review]

**Pull request merged**